pkg/operator/ceph/config/monstore.go

```diff
@@ -62,7 +62,7 @@ type Option struct {
 
 // SetIfChanged sets a config in the centralized mon configuration database if the config has
 // changed value.
-// https://docs.ceph.com/docs/master/rados/configuration/ceph-conf/#monitor-configuration-database
+// https://docs.ceph.com/en/latest/rados/configuration/ceph-conf/#monitor-configuration-database
```

Comment:
Were these links broken?

Reply:
Yes, `docs/master` was broken when we tried to access them. I see it now redirects to `en/latest`. Shall we revert this change?

Reply:
a89da05bb4